### ISSUE TITLE
Untwine fails to open node on OSX

### ISSUE THREAD

**Issue by hobu** [hobuinc/untwine#123]
Curious failure from https://github.com/qgis/QGIS/issues/48798

This is coming from the `qgis` branch, which is not equal to `main`

```
    frame #9: 0x00000001000f7cec untwine`untwine::epf::Writer::run(this=0x0000000106a3bad0) at Writer.cpp:144:13
    frame #10: 0x0000000100102970 untwine`decltype(__f=0x000060000022d388, __a0=0x000060000022d398)).*fp()) std::__1::__invoke<void (untwine::epf::Writer::*&)(), untwine::epf::Writer*&, void>(void (untwine::epf::Writer::*&)(), untwine::epf::Writer*&) at type_traits:3859:1
    frame #11: 0x00000001001028f4 untwine`std::__1::__bind_return<void (untwine::epf::Writer::*)(), std::__1::tuple<untwine::epf::Writer*>, std::__1::tuple<>, __is_valid_bind_return<void (untwine::epf::Writer::*)(), std::__1::tuple<untwine::epf::Writer*>, std::__1::tuple<> >::value>::type std::__1::__apply_functor<void (__f=0x000060000022d388, __bound_args=size=1, (null)=__tuple_indices<0> @ 0x00000001703fed7f, __args=size=0)(), std::__1::tuple<untwine::epf::Writer*>, 0ul, std::__1::tuple<> >(void (untwine::epf::Writer::*&)(), std::__1::tuple<untwine::epf::Writer*>&, std::__1::__tuple_indices<0ul>, std::__1::tuple<>&&) at bind.h:257:12
    frame #12: 0x00000001001028ac untwine`std::__1::__bind_return<void (untwine::epf::Writer::*)(), std::__1::tuple<untwine::epf::Writer*>, std::__1::tuple<>, __is_valid_bind_return<void (untwine::epf::Writer::*)(), std::__1::tuple<untwine::epf::Writer*>, std::__1::tuple<> >::value>::type std::__1::__bind<void (this=0x000060000022d388)(), untwine::epf::Writer*>::operator()<>() at bind.h:292:20
    frame #13: 0x0000000100102868 untwine`decltype(__f=0x000060000022d388)(), untwine::epf::Writer*>&>(fp)()) std::__1::__invoke<std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*>&>(std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*>&) at type_traits:3918:1
    frame #14: 0x0000000100102820 untwine`void std::__1::__invoke_void_return_wrapper<void, true>::__call<std::__1::__bind<void (__args=0x000060000022d388)(), untwine::epf::Writer*>&>(std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*>&) at invoke.h:61:9
    frame #15: 0x00000001001027f8 untwine`std::__1::__function::__alloc_func<std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*>, std::__1::allocator<std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*> >, void ()>::operator(this=0x000060000022d388)() at function.h:178:16
    frame #16: 0x0000000100101594 untwine`std::__1::__function::__func<std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*>, std::__1::allocator<std::__1::__bind<void (untwine::epf::Writer::*)(), untwine::epf::Writer*> >, void ()>::operator(this=0x000060000022d380)() at function.h:352:12
    frame #17: 0x000000010017bb80 untwine`std::__1::__function::__value_func<void ()>::operator(this=0x00000001703feef8)() const at function.h:505:16
    frame #18: 0x0000000100177a10 untwine`std::__1::function<void ()>::operator(this= Function = untwine::epf::Writer::run() )() const at function.h:1182:12
    frame #19: 0x0000000100177724 untwine`untwine::ThreadPool::work(this=0x0000000106a3bae8) at ThreadPool.cpp:93:17
    frame #20: 0x000000010017a2e0 untwine`untwine::ThreadPool::go(this=0x0000600000016608)::$_0::operator()() const at ThreadPool.cpp:50:43
    frame #21: 0x000000010017a27c untwine`decltype(__f=0x0000600000016608)::$_0>(fp)()) std::__1::__invoke<untwine::ThreadPool::go()::$_0>(untwine::ThreadPool::go()::$_0&&) at type_traits:3918:1
    frame #22: 0x000000010017a218 untwine`void std::__1::__thread_execute<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct> >, untwine::ThreadPool::go()::$_0>(__t=size=2, (null)=__tuple_indices<> @ 0x00000001703fef7f)::$_0>&, std::__1::__tuple_indices<>) at thread:287:5
    frame #23: 0x0000000100179a04 untwine`void* std::__1::__thread_proxy<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct> >, untwine::ThreadPool::go()::$_0> >(__vp=0x0000600000016600) at thread:298:5
```

**Comment by hobu** [comment]
I think this is now fixed by 4e60f8a7f5edd094d4e5d44e7093677e841a1213 please test and confirm.

**Comment by abellgithub** [comment]
Closing due to non-response. Please reopen if you have more information.
